apidoc/Titanium/Media/VideoPlayer.yml

```diff
@@ -685,6 +717,37 @@ properties:
     platforms: [iphone, ipad]
     type: MovieSize
 
+  - name: overlayView
+    summary: Use the overlay view to add additional custom views between the video content and the controls.
+    description: |
+        Note that the overlay view must be sized explicitly: percentage sizes do not work properly.
+
+        The following code sample creates a video player with a background image:
+
+            var videoPlayer = Ti.Media.createVideoPlayer({
+                height : 300,
+                width : 300,
+                overlayView : Ti.UI.createView({
+                    backgroundImage: 'videoPlayerBG.png
+                    width : 300,
+                    height : 300
+                }),
+                borderRadius : 20,
+                borderWidth : 2,
+                borderColor : 'blue',
+            });
+    platforms: [iphone, ipad]
+    type: Titanium.UI.View
+
+  - name: pictureInPictureEnabled
+    summary: Whether or not the receiver allows Picture in Picture playback
+    platforms: [iphone, ipad]
+    type: Boolean
+    default: true
+    osver:
+        ios:
+            max: "9.0"
```

Comment:
I am wondering how this passes linting. Some notes:

- It should be "min", not "max", because it was *introduced* in [iOS 9](https://developer.apple.com/documentation/avkit/avplayerviewcontroller/1615821-allowspictureinpictureplayback)
- The syntax is `    osver: {ios: {min: "9.0"}}`
- Add a `.` after `playback`